functions-methods.js

```diff
@@ -20,7 +25,27 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail(email) {
+    let emailType = null;
+
+    switch (getEmailDomain(email)) {
+        case 'novi.nl':
+            emailType = 'Medewerker';
+            break;
+        case 'novi-education.nl':
+            emailType = 'Student';
+            break;
+        default :
+            emailType = 'Extern'
+    }
+
+    return emailType;
```

Comment:
Als je na het switch statement niets meer wilt doen, dan kun je ook meteen de waarde teruggeven en hoef je die ook niet in een variabele op te slaan:
```suggestion
    switch (getEmailDomain(email)) {
        case 'novi.nl':
            return 'Medewerker';
        case 'novi-education.nl':
            return 'Student';
        default :
            return 'Extern'
    }
```